EKSD_LATEST_RELEASES

```diff
@@ -3,15 +3,15 @@ releases:
   number: 17
   kubeVersion: v1.18.20
```

Comment:
Going to get rid of 18 here?

Reply:
I think we actually need to keep 18 because 19 uses the 18 pause image